command/sub8_launch/launch/subsystems/passive_sonar.launch

```diff
@@ -1,10 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
 <launch>
-  <node pkg="sub8_launch" type="passive_sonar_conn" name="passive_sonar_conn" />
-  <node pkg="mil_passive_sonar" type="hydrophones" name="hydrophones_heading">
-    <param name="dist_h" value="2.286e-02" />
-    <param name="dist_h4" value="2.286e-02" />
-    <param name="v_sound" value="1482" />
-  </node>
+  <arg name="environment" default="real" />
+  <group if="$(eval environment == 'real')">
+    <node pkg="sub8_launch" type="passive_sonar_conn" name="passive_sonar_conn" />
+    <node pkg="mil_passive_sonar" type="hydrophones" name="hydrophones_heading">
+      <param name="dist_h" value="2.286e-02" />
```

Comment:
Note that this will not launch the passive_sonar_conn and hydrophones nodes with the main launch. Be sure to test on the sub before merging